### PR TITLE
Initial code

### PR DESCRIPTION
**Pull request by perry**

### CODE REVIEW

**Comment by perry** [comment]
@incuna/js fancy taking a quick look over this?

**Comment by maxpeterson** [comment]
Looks reasonable

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "angular-sticky-footer",
+  "version": "0.0.1",
+  "dependencies": {
+    "angular": "~1.2",
+
```
Comment (henrahmagix) [inline bower.json:6]: Weird empty line

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "jquery": "~1.10"
```
Comment (henrahmagix) [inline bower.json]: Surely this version can be `*`, or > whatever version first had both `outerHeight` and `parents()`.

Comment (henrahmagix) [inline bower.json]: This will work with jQuery 2, so that should be allowed.

File: bower.json (hunk 1 above)
Comment (perry) [inline bower.json:6]: Convention of separating types of dependencies.

File: bower.json (hunk 2 above)
Comment (perry) [inline bower.json]: Forgot about jQuery 2. I'll change it to `>1.10` for now.

**Comment by perry** [comment]
Bump

**Comment by henrahmagix** [comment]
:+1:

**Comment by perry** [comment]
Ta

**Pull request merged**